modules/Bio/EnsEMBL/Variation/VCFCollection.pm

```diff
@@ -632,13 +632,19 @@ sub get_all_Alleles_by_VariationFeature {
     my $info = $vcf->get_info;
     my $vcf_alts = $vcf->get_alternatives;
     my %allele_map = map {$_->{b_index} => $_->{a_allele}} @$matched;
-
-    foreach my $pop(@pops) {
+     POP : {
+     foreach my $pop(@pops) {
 
       # this allows for an empty population name to be looked up as e.g. AF
       my $raw_name = exists($pop->{_raw_name}) ? $pop->{_raw_name} : $pop->name;
       my $suffix = $raw_name ? '_'.$raw_name : '';
       my $pop_id = $pop->dbID;
+      foreach my $allele (values %allele_map) {
```

Comment:
A comment would be useful here to explain this handles multiple lines with the same alt by only considering the first